xbmc/interfaces/legacy/ListItem.h

```diff
@@ -333,6 +333,17 @@ namespace XBMCAddon
       String getProperty(const char* key);
 
       /**
+       * getArt(key) -- Returns a listitem art value as a string, similar to an infolabel.\n
+       * \n
+       * key            : string - art name.\n
+       * \n
+       *
+       * example:
+       *   - poster = self.list.getSelectedItem().getArt('poster')
+       */
+      String getArt(const char* key);
```

Comment:
Could use some better documentation. 'It returns a value'  A value of what? And list the available types please, as setArt does.